### DISCUSSION TITLE
netbox (v3.5.6) - container shows status warning

### DISCUSSION THREAD

**Discussion by sadden3194** [netbox-community/netbox#13315] [Help Wanted!]
Hi all!

My container shows the status warning.
In the log files I cant see any errors or warnings.
any idea what I can check to solve this issue?

maybe this helps?

![image](https://github.com/netbox-community/netbox/assets/17011498/41f73928-79bf-4814-860a-e406fb09923a)

**Comment by candlerb** [comment]
> My container shows the status warning

What container? A docker container? Running on what platform? Configured how? What exact warning message do you see?

Where are those access logs from? The format `GET / => generated 143 bytes ... (1 switches on core 0)` is not familiar to me; Googling suggests it could be nginx uWSGI, [here](https://stackoverflow.com/questions/45197267/what-does-this-uwsgi-output-mean-x-switches-on-core-0).

Those logs could be test accesses from 172.18.0.1 twice per minute, and that looks like a Docker engine assigned IP.  The response code 400 is an error (I would expect Netbox to return code 302, redirect to `/login/?next=/`).  If you look at the request/response body, e.g. with tcpdump, that might give you some more clues.

But if it is working, and your only problem is that you see a "status warning", then the starting point should be to show exactly what that warning says.

**Comment by sadden3194** [comment]
I am unsing the docker container on my synology NAS.
The logs are from the container manager (synology package)

**Reply by candlerb** [reply]
If by "the docker container" you mean the image published as [netbox-docker](https://github.com/netbox-community/netbox-docker/),  then this is not sufficient to run Netbox by itself: you need Postgres and Redis, and you also need to run containers for the rq-worker and housekeeping tasks.

You can manually set up these dependencies and patch them together using environment variables, but normally you'd handle this using [docker-compose](https://github.com/netbox-community/netbox-docker/blob/release/docker-compose.yml).

If you mean some other all-in-one Netbox docker container you've found, then please say what it is.

Also, you still haven't said what this "warning" message you see is.

**Comment by sadden3194** [comment]
![image](https://github.com/netbox-community/netbox/assets/17011498/a423e0fa-5c77-4957-82c3-e1871c566974)

![image](https://github.com/netbox-community/netbox/assets/17011498/0f1b7e9a-abf7-41ce-8a2a-2be0cba233ad)

I only see this.

**Reply by ziggekatten** [reply]
You should post your question on Linuxserver.io, as you use their image, and I suppose their compose file. Issues can be related to their specific build.

**Reply by candlerb** [reply]
And/or on a forum for the Synology container manager, to find out how to expand that orange dot into a more useful status message.
